### PR TITLE
📦 NEW:  limit functionality to cli

### PR DESCRIPTION
**Pull request by firasdib**
Implements the feature described in https://github.com/ahmadawais/corona-cli/issues/16

### CODE REVIEW

**Review by viditvarshney — APPROVED** [review]
Good improvement

**Review by Inukares — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ const {
 } = require("./utils/table.js");
 const xcolor = cli.flags.xcolor;
 const sortBy = cli.flags.sort;
+const limit = cli.flags.limit;
```
Comment (firasdib) [inline index.js]: If the limit is negative, some unexpected results might be generated: `slice(0, -10)` for example might be a bit odd. Should we limit this to positive integers? Is it possible to do this in the `cli` utility?

**Comment by ahmadawais** [comment]
Definitely, we should only use absolute positive values.

>

File: index.js (hunk 1 above)
Comment (firasdib) [reply index.js]: How do you propose we solve this @ahmadawais? Should we default to a given value if the number is negative, or should we throw an error? Can this be handled in the CLI utility?

**Comment by ahmadawais** [comment]
I suggest using something like
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs
Just turn the negative value to positive one.

>

**Comment by ahmadawais** [comment]
Thanks for the PR. Can you please update the PR in light of the latest release.

🔗 https://github.com/ahmadawais/corona-cli/releases/tag/3.4.0
🔗 https://www.npmjs.com/package/corona-cli

**Comment by firasdib** [comment]
Looks like some pretty major changes were made, and I'm not sure I'm the right person to fix this, even though I did the changes...

**Comment by firasdib** [comment]
I gave it a go. Have a look and see if you agree with the changes.

**Comment by ahmadawais** [comment]
Oh dang, again needs to be rebased.

**Comment by firasdib** [comment]
Rebased again..

**Review by ahmadawais — APPROVED** [review]
Looks good to me! ✔︎

**Comment by ahmadawais** [comment]
No worries, I had rebased them on my end already.

**Comment by ahmadawais** [comment]
Thank you!

🔗 https://github.com/ahmadawais/corona-cli/releases/tag/3.6.0
🔗 https://www.npmjs.com/package/corona-cli

I chose `-l` as the alias.

**Comment by peschee** [comment]
The help text is off:

```
   --limit, -n          Limit output to N entries
```

The alias is still displayed as `-n`, `-l` works.

**Comment by ahmadawais** [comment]
Dang it. I thought I had removed it.

🔗 https://github.com/ahmadawais/corona-cli/releases/tag/3.8.4
🔗 https://www.npmjs.com/package/corona-cli

npm i -g corona-cli

>

**Pull request merged**